packages/server/README.md

```diff
@@ -13,7 +15,7 @@ You can use Apollo Server as:
 * The GraphQL server for a [subgraph](https://www.apollographql.com/docs/federation/subgraphs/) in a federated supergraph
 * The gateway for a [federated supergraph](https://www.apollographql.com/docs/federation/)
 
-Apollo Server provides a simple API for integrating with any Node.js web framework or serverless environment. The `@apollo/server` package itself ships with a standalone zero-dependency web server, and a middleware implementation for [the Express web framework](https://expressjs.com/). Integrations with other environments are community-maintained.
+Apollo Server provides a simple API for integrating with any Node.js web framework or serverless environment. The `@apollo/server` package itself ships with a standalone web server (built using [the Express web framework](https://expressjs.com/)) as well as an Express middleware implementation. Integrations with other environments are community-maintained.
```

Comment:
So we are actually actively masking the fact that this is built with Express at the TS level (the req/res options to context) so i don't know that it's helpful to call it out here? Like I want it to be backwards-compatible to stop using Express and just use raw Node HTTP if we wanted. (I looked into replacing `express` with `compose-middleware` + `finalhandler` yesterday, it just seemed a bit pointless.)